docs/dev/design_documents/client_release_cadence.md

```diff
@@ -0,0 +1,33 @@
+# Chef Release Cadence
```

Comment:
Just occurred to me - shouldn't these Chef references be Chef Infra references?

Reply:
yea we should probably fix them as we see them. I figured the only ones that were a real priority where those in the main readme.